routes/cube_routes.js

```diff
@@ -3160,6 +3165,65 @@ router.post('/api/updatecards/:id', ensureAuth, function(req, res) {
   });
 });
 
+function maybeCards(cube) {
+  const maybe = (cube.maybe || []).filter((card) => card.cardID);
```

Comment:
are there negative or 0 cardIDs you're filtering out? do those just exist in the cube?

Reply:
No, just trying to avoid server crashes if something gets corrupted.